src/software/ai/hl/stp/tactic/move_to_goal_line/move_to_goal_line_tactic.cpp

```diff
@@ -14,12 +14,14 @@ void MoveToGoalLineTactic::updateWorldParams(const World &world) {}
 double MoveToGoalLineTactic::calculateRobotCost(const Robot &robot,
                                                 const World &world) const
 {
-    // Prefer robots closer to the goal line
-    // We normalize with the total field length so that robots that are within the
-    // field have a cost less than 1
-    double cost = (robot.position() - world.field().friendlyGoalCenter()).length() /
-                  world.field().totalXLength();
-    return std::clamp<double>(cost, 0, 1);
+    if (world.friendlyTeam().getGoalieId() == robot.id())
```

Comment:
This cost function is a problem given the name of the tactic, since we wouldn't expect the goalie to always be assigned to a tactic with a generic name like that

Reply:
renamed the tactic to move_goalie_to_goal_line